biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -407,9 +429,34 @@ real(r8) :: tcwood             ! Temperature response function for wood
                               
                               ! Scale for leaf nitrogen profile
                               nscaler = exp(-kn(ft) * cumulative_lai)
+                              
+                              ! Leaf maintenance respiration to match the base rate used in CN
+                              ! but with the new temperature functions for C3 and C4 plants.
+
+                              ! CN respiration has units:  g C / g N [leaf] / s. This needs to be
+                              ! converted from g C / g N [leaf] / s to umol CO2 / m**2 [leaf] / s
+                              
+                              ! Then scale this value at the top of the canopy for canopy depth
+                              ! Leaf nitrogen concentration at the top of the canopy (g N leaf / m**2 leaf)
+                              select case(hlm_parteh_model)
```

Comment:
add some notes on why there are these different calculations of lnc_top.

Reply:
@rosiealice , can you give me a little more guidance on what you would like to see here?